gr-qtgui/lib/waterfall_sink_c_impl.cc

```diff
@@ -333,8 +333,8 @@ void waterfall_sink_c_impl::windowreset()
 void waterfall_sink_c_impl::buildwindow()
 {
     d_window.clear();
-    if (d_wintype != filter::firdes::WIN_NONE) {
-        d_window = filter::firdes::window(d_wintype, d_fftsize, 6.76);
+    if (d_wintype != fft::window::WIN_NONE) {
```

Comment:
shouldn't need the 6.76 magic number hardcoded here since it is the default

Reply:
Hah yeah! That's why I submitted 3adb522b0ddf6736605beea797016353a1d4a2b8 then forgot about it.

Reply:
Fixed that in all 4 places.